control/rlocus.py

```diff
@@ -232,16 +232,11 @@ def root_locus(sys, kvect=None, xlim=None, ylim=None,
             ax.set_ylim(ylim)
 
         # Draw the grid
-        if grid and sisotool:
-            if isdtime(sys, strict=True):
-                zgrid(ax=ax)
-            else:
-                _sgrid_func(fig=fig)
-        elif grid:
+        if grid:
             if isdtime(sys, strict=True):
                 zgrid(ax=ax)
             else:
-                _sgrid_func()
+                _sgrid_func(fig=fig if sisotool else None)
```

Comment:
Any idea why there are two different sgrid functions? there is this one and there is grid.sgrid. Maybe a different PR, but would be good to unify these and get the best of each of them.

Reply:
Apparently _sgrid_func() was removed in #193 but reintroduced through some merge gone wrong?

Reply:
908fabebacfe642479f2b75a3a40f014a87a13ef